pyrail/irail.py

```
@@ -176,12 +180,12 @@ def do_request(self, method: str, args: Optional[Dict[str, Any]] = None) -> Opti
         request_headers: Dict[str, str] = self._add_etag_header(method)
 
         try:
-            response = session.get(url, params=params, headers=request_headers)
+            response = await session.get(url, params=params, headers=request_headers)
```

Ruff (0.8.2)

Undefined name `session`

(F821)